### PR TITLE
Documentation page

### PR DESCRIPTION
**Pull request by fedorgrab**
The doc page is pretty simple right now, but I will add more info to it. The client tool has a simple interface, so there's not much stuff to document yet.

You can check the doc page by this link:

https://cellarium-ai.github.io/cellarium-cas/

### CODE REVIEW

File: docs/source/modules/installation.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Installation
+############
+
+
+From github repository
+++++++++++++++++++++++
+
+Install latest alpha version using `pip` with a specified version::
+
+    pip install git+https://github.com/broadinstitute/cellarium-cas.git@1.0a1
```
Comment (ordabayevy) [inline docs/source/modules/installation.rst:12]: Fix the org in the link to `cellarium-ai`

File: docs/source/modules/usage.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
++++++++++++++++++++++++++++
+You can annotate 10x Cell Ranger h5 matrices from local disk::
+
+    response = cas.annotate_10x_h5_file(filepath="your_path_to_local_h5_file.h5")
+
+Anndata File from disk
+++++++++++++++++++++++
+::
+
+    response = cas.annotate_anndata_file(filepath="your_path_to_local_h5_file.h5ad")
```
Comment (ordabayevy) [inline docs/source/modules/usage.rst:34]: Does it support files on GCS?

File: docs/source/modules/workflow.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Token Validation
+++++++++++++++++
+
+**CASClient** first checks the API token validity
+
+Data Validation
++++++++++++++++
+
+Next step is to check the data. At the moment, **CASClient** checks if the matrix corresponds to CAS schema before
```
Comment (ordabayevy) [inline docs/source/modules/workflow.rst:14]: nit: a CAS schema

File: docs/source/modules/workflow.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+Depending on the previous step it would next do one of the following:
+
+    1. Fill missing features with zeros.
+    2. Omit extra features.
+
+Data Transfer to Backend
+++++++++++++++++++++++++
+**CASClient** splits input data matrices into smaller shards and asynchronously send them to backend to achieve the
+maximum speed of processing
```
Comment (ordabayevy) [inline docs/source/modules/workflow.rst:28]: nit: processing.

File: docs/source/modules/workflow.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
++++++++++++++++++
+
+Depending on the previous step it would next do one of the following:
+
+    1. Fill missing features with zeros.
+    2. Omit extra features.
+
+Data Transfer to Backend
+++++++++++++++++++++++++
+**CASClient** splits input data matrices into smaller shards and asynchronously send them to backend to achieve the
```
Comment (ordabayevy) [inline docs/source/modules/workflow.rst:27]: nit: the backend
nit: The CASClient

File: docs/source/modules/workflow.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Data Sanitization
++++++++++++++++++
+
+Depending on the previous step it would next do one of the following:
+
+    1. Fill missing features with zeros.
+    2. Omit extra features.
+
+Data Transfer to Backend
```
Comment (ordabayevy) [inline docs/source/modules/workflow.rst:25]: nit: the Backend

File: docs/source/modules/usage.rst [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+    api_token = "api_token_string_goes_here"
+    cas = CASClient(api_token=api_token)
+
+Annotation
+----------
+
+10x Cell Ranger h5 matrices
++++++++++++++++++++++++++++
+You can annotate 10x Cell Ranger h5 matrices from local disk::
```
Comment (ordabayevy) [inline docs/source/modules/usage.rst:26]: nit: the local disk

File: docs/source/modules/usage.rst [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+    response = cas.annotate_10x_h5_file(filepath="your_path_to_local_h5_file.h5")
+
+Anndata File from disk
```
Comment (ordabayevy) [inline docs/source/modules/usage.rst:30]: nit: the disk

**Review by ordabayevy — CHANGES_REQUESTED** [review]

File: .github/actions/docs/action.yaml [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+      shell: bash
+
+    - name: Deploy Sphinx docs
+      uses: sphinx-notes/pages@v3
+      with:
+        documentation_path: ./docs/source/
+        pyproject_extras: docs
```
[the remark this answers is not in the document]
Comment (fedorgrab) [reply .github/actions/docs/action.yaml:20]: It actually works well. I don't exactly know why they put this in the example. 
Here's my workflow:
https://github.com/cellarium-ai/cellarium-cas/actions/runs/6151357996/job/16691284766

**Review by ordabayevy — APPROVED** [review]

**Pull request merged**